python/cuml/cuml/internals/array.py

```diff
@@ -1201,13 +1203,13 @@ def from_input(
                     " utilization."
                 )
 
-        if check_cols:
+        if check_cols is not False:
```

Comment:
This and the equivalent change below are necessary so `check_cols=0`/`check_rows=0` still applies the check.